default.nix

```diff
@@ -3,11 +3,13 @@ let
 in
 { system ? builtins.currentSystem
 , config ? {}
-, gitrev ? localLib.commitIdFromGitRepo ./.git
+, gitrev ? "beta"
```

Comment:
I'm not sure if I understand correctly what this is used for, but currently we are using `icarus-master` as the most up-to-date branch, shouldn't it be changed to that?

Reply:
the label is arbitrary and can be changed to anything you want. The issue with the previous method was that it wouldn't work in a git submodule where the `.git` folder is not available at that level. It also avoids unnecessary rebuilds as a new commit might not necessarily change the build inputs (for example changing the README doesn't require a new rebuild).

Reply:
ping

Reply:
I had misunderstood then, makes sense to me!